mlflow/models/evaluation/default_evaluator.py

```diff
@@ -669,9 +659,14 @@ def _log_model_explainability(self):
             )
             _logger.debug("", exc_info=True)
 
+        def _adjust_color_bar():
+            pyplot.gcf().axes[-1].set_aspect('auto')
+            pyplot.gcf().axes[-1].set_box_aspect(50)
+
         def plot_beeswarm():
             pyplot.subplots_adjust(bottom=0.2, left=0.4)
```

Comment:
Do we need this line?

Reply:
No. Now it is auto size layout . let me remove it.